ci/cleanup/aws.py

```diff
@@ -82,9 +82,7 @@ def clean_up_sqs() -> None:
 
 
 def clean_up_ec2() -> None:
-    print(
-        f"Terminating scratch ec2 instances whose age exceeds the deletion time (default lifespan: {scratch.MAX_AGE})"
```

Comment:
I got rid of the scratch.MAX_AGE line here despite that not really being part of this change because it's just confusing (`bin/scratch` isn't the only script that creates instances, and other ones can in general use whatever timeout they like)